indra/tests/test_statements_serialization.py

```diff
@@ -1,88 +1,99 @@
 from __future__ import absolute_import, print_function, unicode_literals
-from builtins import dict, str
```

Comment:
The issue was that we wanted to enforce a pure Unicode sandwich where everything inside the application was unicode, with bytes only at the edges. Didn't look much into six--would probably have helped make the code cleaner but ultimately wouldn't have been sufficient by itself. See this blog post I put together on a 2/3 compatible unicode sandwich: http://johnbachman.net/building-a-python-23-compatible-unicode-sandwich.html#building-a-python-23-compatible-unicode-sandwich